docs/CONTRIBUTING.rst

```diff
@@ -1,5 +1,89 @@
+
+
+Contributing to the project
+===========================
+
+We strongly advise to use `Virtualenv <https://virtualenv.pypa.io/en/stable/>`_ to install Cekit. Please consult your package manager for the correct package name.
+
+To create custom Python virtual environment please run following commands on your system:
+
+.. code-block:: bash
+
+    # Prepare virtual environment
+    virtualenv ~/cekit
+    source ~/cekit/bin/activate
+
+    # Install Cekit
+    # Execute the same command to upgrade to latest version
+    pip install -U cekit
+
+    # Now you are able to run Cekit
+    cekit --help
+
+It is possible to ask virtualenv to inherit pre-installed system packages thereby reducing the virtualenv to a delta between what is installed and what is required. This is achived by using the flag ``--system-site-packages`` (See `here <https://virtualenv.pypa.io/en/latest/userguide/#the-system-site-packages-option>`_ for further information).
+
+.. note::
+
+   Every time you want to use Cekit you must activate Cekit Python virtual environment by executing ``source ~/cekit/bin/activate``
+
+.. note::
+   For those using ZSH a useful addition is `Zsh-Autoswitch-VirtualEnv <https://github.com/MichaelAquilina/zsh-autoswitch-virtualenv>`_ the use of which avoids the labour of manually creating the virtualenv and activating it each time.
+
+Currently within Fedora 29 almost all the required packages are available as RPMs; non-python packages are:
```

Comment:
Maybe instead of listing these packages we could just rely on the runtime dependency mechanism to suggest them? Maybe tests could use same mechanism?

Reply:
The cekit runtime rpm does/should provide the runtime deps ; currently there is nothing for the dev side - I have provided a set of ansible scripts instead.